reverse-strings/main.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import (
+	"github.com/ivansukach/go-internship/reverse-strings/tools"
+	"golang.org/x/exp/errors/fmt"
+)
+
+func main() {
+	fmt.Println(tools.Reverse("Hello, world!"))
```

Comment:
It there any reason to use fmt from golang.org/x/exp/errors/fmt package instead of default fmt package?